### ISSUE TITLE
Updated Release

### ISSUE THREAD

**Issue by jvanderaa** [napalm-automation/napalm-fortios#68]
###Description of Issue/Question

Release update coming? Over 5 years now??

### Did you follow the steps from https://github.com/napalm-automation/napalm#faq
- [x] Yes
- [ ] No

It looks like there was at least one beneficial update merged in. Can this get updated to Pypi?

**Comment by ktbyers** [comment]
@awlx Are you willing to do this?

If not do you care if @jvanderaa does this?

Regards, Kirk

**Comment by awlx** [comment]
I am pretty low on time at the moment sadly and don't use Fortios napalm myself anymore. So would be better if someone does it who can actually test and maintain.

**Comment by jvanderaa** [comment]
I don't have much experience with Fortinet in general, I've done some. But I can help keep things progressing.

**Comment by shrank** [comment]
looks like the main dependency is no longer maintained:
https://github.com/spotify/pyfg

**Comment by jvanderaa** [comment]
Yeah, that looks like there needs to be some major rework on this to get working.

**Comment by n3s0** [comment]
I'm currently in the process of reworking it on the fork I made of the spotify/pyfg dependency. Currently working out the bgp stuff related to it because Fortinet has made some changes to that...

https://github.com/n3s0/pyfg

I have forked the napalm-fortios and I'm willing to perform a merge after I've performed some more tests.

https://github.com/n3s0/napalm-fortios
